src/recipes/mango-lassi.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "Mango Lassi",
     "description": "This recipe comes from the cookbook author Madhur Jaffrey, who said, 'Mangoes are supposed to be good for the eyes.'",
-    "github": "parammittal16",
+    "github": "parammittal  16",
```

Comment:
Mistake here?

Reply:
Must be, (I wasn't trying to change usernames) I'll fix and recommit.

Reply:
fix implemented with last commit.